### PR TITLE
Django 2.0

### PR DESCRIPTION
**Pull request by hipek8**

### CODE REVIEW

File: src/ralph/data_center/migrations/0033_auto_20240621_1217.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+            name='vip',
+            managers=[
+                ('polymorphic_objects', django.db.models.manager.Manager()),
+            ],
+        ),
+        migrations.AlterField(
+            model_name='baseobjectcluster',
+            name='base_object',
+            field=ralph.lib.mixins.fields.BaseObjectForeignKey(limit_choices_to=ralph.lib.mixins.fields.BaseObjectForeignKey.limit_choices, on_delete=django.db.models.deletion.CASCADE, related_name='clusters', to='assets.BaseObject'),
+        ),
```
Comment (hipek8) [inline src/ralph/data_center/migrations/0033_auto_20240621_1217.py:44]: Write a test

File: src/ralph/data_center/migrations/0034_auto_20240628_1207.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+        ('data_center', '0033_auto_20240621_1217'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='baseobjectcluster',
+            name='base_object',
+            field=ralph.lib.mixins.fields.BaseObjectForeignKey(limit_models=['data_center.Database', 'data_center.DataCenterAsset', 'virtual.CloudHost', 'virtual.VirtualServer'], on_delete=django.db.models.deletion.CASCADE, related_name='clusters', to='assets.BaseObject'),
+        ),
+    ]
```
Comment (hipek8) [inline src/ralph/data_center/migrations/0034_auto_20240628_1207.py:20]: Write a test

File: src/ralph/licences/migrations/0008_auto_20240628_1207.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+        ('licences', '0007_auto_20240506_1633'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='baseobjectlicence',
+            name='base_object',
+            field=ralph.lib.mixins.fields.BaseObjectForeignKey(limit_models=['back_office.BackOfficeAsset', 'data_center.DataCenterAsset', 'virtual.VirtualServer', 'data_center.Cluster'], on_delete=django.db.models.deletion.CASCADE, related_name='licences', to='assets.BaseObject', verbose_name='Asset'),
+        ),
+    ]
```
Comment (hipek8) [inline src/ralph/licences/migrations/0008_auto_20240628_1207.py:20]: Write a test

File: src/ralph/supports/migrations/0010_auto_20240628_1207.py [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+        ('supports', '0009_auto_20240506_1633'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='baseobjectssupport',
+            name='baseobject',
+            field=ralph.lib.mixins.fields.BaseObjectForeignKey(limit_models=['back_office.BackOfficeAsset', 'data_center.DataCenterAsset'], on_delete=django.db.models.deletion.CASCADE, related_name='supports', to='assets.BaseObject', verbose_name='Asset'),
+        ),
+    ]
```
Comment (hipek8) [inline src/ralph/supports/migrations/0010_auto_20240628_1207.py:20]: Write a test

File: src/ralph/tests/migrations/0010_auto_20240621_1217.py [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+        ('tests', '0009_auto_20240621_1009'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='baseobjectforeignkeymodel',
+            name='base_object',
+            field=ralph.lib.mixins.fields.BaseObjectForeignKey(limit_choices_to=ralph.lib.mixins.fields.BaseObjectForeignKey.limit_choices, on_delete=django.db.models.deletion.CASCADE, to='assets.BaseObject'),
+        ),
+    ]
```
Comment (hipek8) [inline src/ralph/tests/migrations/0010_auto_20240621_1217.py:20]: Write a test

File: src/ralph/tests/migrations/0011_auto_20240621_1231.py [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+        ('tests', '0010_auto_20240621_1217'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='baseobjectforeignkeymodel',
+            name='base_object',
+            field=ralph.lib.mixins.fields.BaseObjectForeignKey(limit_choices_to=ralph.lib.mixins.fields.BaseObjectForeignKey.limit_choices, on_delete=django.db.models.deletion.CASCADE, to='assets.BaseObject'),
+        ),
+    ]
```
Comment (hipek8) [inline src/ralph/tests/migrations/0011_auto_20240621_1231.py:20]: Write a test

File: src/ralph/tests/migrations/0012_auto_20240621_1355.py [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+        ('tests', '0011_auto_20240621_1231'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='baseobjectforeignkeymodel',
+            name='base_object',
+            field=ralph.lib.mixins.fields.BaseObjectForeignKey(limit_choices_to=ralph.lib.mixins.fields.BaseObjectForeignKey.limit_choices, on_delete=django.db.models.deletion.CASCADE, to='assets.BaseObject'),
+        ),
+    ]
```
Comment (hipek8) [inline src/ralph/tests/migrations/0012_auto_20240621_1355.py:20]: Write a test

File: src/ralph/tests/migrations/0013_auto_20240627_1512.py [hunk 8]
```diff
@@ -0,0 +1,20 @@
...
+        ('tests', '0012_auto_20240621_1355'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='baseobjectforeignkeymodel',
+            name='base_object',
+            field=ralph.lib.mixins.fields.BaseObjectForeignKey(limit_choices_to=ralph.lib.mixins.fields.BaseObjectForeignKey.limit_choices, on_delete=django.db.models.deletion.CASCADE, to='assets.BaseObject'),
+        ),
+    ]
```
Comment (hipek8) [inline src/ralph/tests/migrations/0013_auto_20240627_1512.py:20]: Write a test

File: src/ralph/tests/migrations/0014_auto_20240628_1213.py [hunk 9]
```diff
@@ -0,0 +1,20 @@
...
+        ('tests', '0013_auto_20240627_1512'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='baseobjectforeignkeymodel',
+            name='base_object',
+            field=ralph.lib.mixins.fields.BaseObjectForeignKey(limit_models=['back_office.BackOfficeAsset', 'data_center.DataCenterAsset'], on_delete=django.db.models.deletion.CASCADE, to='assets.BaseObject'),
+        ),
+    ]
```
Comment (hipek8) [inline src/ralph/tests/migrations/0014_auto_20240628_1213.py:20]: Write a test

File: src/ralph/accessories/tests/factories.py [hunk 10]
```diff
@@ -0,0 +1,32 @@
+from factory import post_generation, Sequence, SubFactory
+from factory.django import DjangoModelFactory
+
+from ralph.accessories.models import Accessory, AccessoryStatus, AccessoryUser
+from ralph.accounts.tests.factories import RegionFactory, UserFactory
+from ralph.assets.tests.factories import CategoryFactory, ManufacturerFactory
+from ralph.back_office.tests.factories import WarehouseFactory
+
+
+class AccessoryFactory(DjangoModelFactory):
```
Comment (hipek8) [inline src/ralph/accessories/tests/factories.py:10]: This is different than data center Accessory. 
It wasn't tested at all.
There is also no API

File: src/ralph/accounts/api.py [hunk 11]
```diff
@@ -72,6 +72,7 @@ class RalphUserViewSet(RalphAPIViewSet):
 class RegionSerializer(RalphAPISerializer):
     class Meta:
         model = Region
+        fields = "__all__"
```
Comment (hipek8) [inline src/ralph/accounts/api.py:75]: All serializers now need field "fields" or "exclude"

File: src/ralph/admin/tests/tests_functional.py [hunk 12]
```diff
@@ -216,10 +216,10 @@ class ChangeListTest(TestCase):
 
     def setUp(self):
         super().setUp()
-        manufacturer = Manufacturer.objects.create(
+        manufacturer = TestManufacturer.objects.create(
```
Comment (hipek8) [inline src/ralph/admin/tests/tests_functional.py:219]: There used to be 2 Manufacturer models. The model used in tests has been renamed to TestManufacturer

File: src/ralph/assets/api/serializers.py [hunk 13]
```diff
@@ -452,12 +466,13 @@ class DCHostSerializer(ComponentSerializerMixin, BaseObjectSerializer):
     class Meta:
         model = DCHost
         fields = [
-            'id',
-            'url',
+            'id', 'url',
```
Comment (hipek8) [inline src/ralph/assets/api/serializers.py]: fixme

**Review by matyldv — APPROVED** [review]

**Pull request merged**